test/performance/scenario4.py

```diff
@@ -0,0 +1,40 @@
+import json
+from locust import HttpUser, task, LoadTestShape, FastHttpUser, tag, constant
+import config as config
+
+
+class lastWaitTime:
+    def __init__(self):
+        self.wait_time = 0
+
+    def add_last_wait_time(self, add_wait_time: int):
+        self.wait_time += add_wait_time
+
+    def get_last_wait_time(self):
+        return self.wait_time
+
+
+last_wait_time = lastWaitTime()
+
+
+def increasing(wait_time):
+    def wait_time_func(self):
+        if last_wait_time.wait_time == 0:
+            last_wait_time.add_last_wait_time(wait_time)
+            return 0
+        return last_wait_time.wait_time
+
+    return wait_time_func
+
+
+class FiveHundredDest(FastHttpUser):
+    wait_time = increasing(30)
```

Comment:
This class level property is set, but doesn't appear to be used in the class? is increasing() required?